### ISSUE TITLE
Tasks for new spring boot versions: 2.5.11 and 2.6.4

### ISSUE THREAD

**Issue by chenrujun** [Azure/azure-sdk-for-java#27863]
Refs: https://github.com/Azure/azure-sdk-for-java/wiki/Spring-Cloud-Azure-tasks-for-a-new-Spring-Boot-release

- Before azure-sdk-for-java in band release
  - [x] Create a new issue to track the following tasks. Sample: [#27863](https://github.com/Azure/azure-sdk-for-java/issues/27863).
  - [ ] Read spring boot release notes. Sample: [Spring-Boot-2.6-Release-Notes](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.6-Release-Notes). Write down items we(Spring Cloud Azure team members) should care about. Just write these items by adding a new comment in the GitHub issue created in previous step.
  - [ ] Upgrade external dependencies' version according to spring-boot-dependencies and spring-cloud-dependencies. Refer to [related readme](https://github.com/Azure/azure-sdk-for-java/blob/aec4c6247ba7ba4de57dd866e3f5511ca4fbd387/sdk/spring/scripts/README.md). **NOTE**: This step should be finished before `Core code complete`. You can get more information about `Core code complete` in the Email with subject like `APRIL Release Kickoff`.

- After azure-sdk-for-java in band release.

  Update spring-boot and spring-cloud version in these places:

  - [ ] [azure-spring-boot-samples](https://github.com/Azure-Samples/azure-spring-boot-samples)
  - [ ] [/sdk/spring/CHANGELOG](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring).
  - [ ] [Reference doc](https://github.com/microsoft/spring-cloud-azure/tree/docs/docs/src/main/asciidoc).
  - [ ] [MS docs](https://docs.microsoft.com/en-us/azure/developer/java/spring-framework/).
  - [ ] [Wiki page: Spring Cloud Azure Timeline](https://github.com/Azure/azure-sdk-for-java/wiki/Spring-Cloud-Azure-Timeline)
  - [ ] [Wiki page: Spring Versions Mapping](https://github.com/Azure/azure-sdk-for-java/wiki/Spring-Versions-Mapping)
  - [ ] [Wiki page: Spring Support Multiple Versions](https://github.com/Azure/azure-sdk-for-java/wiki/Spring-Support-Multiple-Versions)

**Comment by chenrujun** [comment]
Closing this issue in favor of #27989
